exporters/otlp/recordable_test.cc

```diff
@@ -0,0 +1,73 @@
+#include "exporters/otlp/recordable.h"
+
+#include <gtest/gtest.h>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace otlp
+{
+TEST(Recordable, SetIds)
+{
+  Recordable rec;
```

Comment:
nit: It would be better to move this definition to line 25, so Recordable is defined close to where it is used.